### PR TITLE
Docs: Fix some phrasing, add links, translation

### PR DESCRIPTION
**Pull request by candideu**

### CODE REVIEW

**Review by lizheming — APPROVED** [review]

File: docs/src/en/guide/get-started/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Welcome to Waline. In just a few steps, you can enable Waline to provide comment
 
    ::: note
 
-   If you haven't logined, we recommend you to sign in with GitHub.
+   If you haven't logged ine, we recommend you to sign in with GitHub.
```
Comment (Mister-Hope) [inline docs/src/en/guide/get-started/README.md:31]: I think there is a extra e

File: docs/src/en/guide/get-started/README.md [hunk 2]
```diff
@@ -79,14 +79,17 @@ Welcome to Waline. In just a few steps, you can enable Waline to provide comment
 
 ## Importing in HTML (Client)
 
-Make the following settings on your web page:
+Here is how you can implement Waline on your web page:
```
Comment (Mister-Hope) [inline docs/src/en/guide/get-started/README.md:82]:
```diff
- Here is how you can implement Waline on your web page:
+ Here is how you can import Waline on your web page:
```

Comment (Mister-Hope) [reply docs/src/en/guide/get-started/README.md:82]: I am not sure, but I do think `implement` is not better, that sound like achieve something, but waline is just tool.

Anyway, these improvements are nice, and we welcome you to improve other pages if you have time.

File: docs/src/en/guide/get-started/README.md (hunk 1 above)
Comment (candideu) [reply docs/src/en/guide/get-started/README.md:31]: There is, you can remove it. Thanks for flagging

File: docs/src/en/guide/get-started/README.md (hunk 2 above)
Comment (candideu) [reply docs/src/en/guide/get-started/README.md:82]: You can use "add", "embed", "import", etc. Whatever makes the most sense to you.

**Pull request merged**